package.json

```diff
@@ -90,7 +91,8 @@
     "tar": "^6.2.1",
     "uuid": "^9.0.1",
     "xml-js": "^1.6.11",
-    "yargs": "^17.7.2"
+    "yargs": "^17.7.2",
+    "hosted-git-info": "^7.0.2"
```

Comment:
Let's not introduce this package and change the format in this PR. We can drop any non-compliant urls, maybe with a log in DEBUG_MODE.